omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,61 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',function ($scope,TagService,ngDialog) {
+    $scope.init = function(personUuid) {
+            $scope.thisPersonUuid = personUuid;
+          $scope.loadTags =function () {
+              var payload = {
+                  objectType: 'org.openmrs.Person',
+                  objectUuid: personUuid
+              };
+              TagService.getTags(payload).then(function (res) {
+                  $scope.tags = res;
+              });
+          }
+         $scope.loadTags();
```

Comment:
Sorry it was my oversight, the call is outside of the loadTags function so it's not recursive, can you please move loadTags out of the init function?